nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -589,6 +589,9 @@ The module update takes care of the new config syntax and the data itself (user
 
 - `python3.pkgs.flitBuildHook` has been removed. Use `flit-core` and `format = "pyproject"` instead.
 
+-  Now `magma` defaults to `magma-hip` instead of `magma-cuda`. It also
+   respects the `config.cudaSupport` and `config.rocmSupport` options.
```

Comment:
@mweinelt does this ad work?

Reply:
Sounds good to me. Ultimately the release editors will reword things if necessary.